addOns/exim/src/test/java/org/zaproxy/addon/exim/har/HarImporterUnitTest.java

```diff
@@ -22,41 +22,79 @@
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.equalTo;
 import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.is;
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.BDDMockito.given;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.verify;
 
-import edu.umass.cs.benchlab.har.HarEntries;
-import edu.umass.cs.benchlab.har.HarLog;
+import de.sstoehr.harreader.HarReader;
+import de.sstoehr.harreader.model.HarEntry;
+import de.sstoehr.harreader.model.HarLog;
 import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
 import java.nio.file.Path;
+import java.util.ArrayList;
 import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.function.Consumer;
+import org.apache.logging.log4j.Level;
+import org.apache.logging.log4j.core.LogEvent;
+import org.apache.logging.log4j.core.LoggerContext;
+import org.apache.logging.log4j.core.StringLayout;
+import org.apache.logging.log4j.core.appender.AbstractAppender;
+import org.apache.logging.log4j.core.config.Configurator;
+import org.apache.logging.log4j.core.config.LoggerConfig;
+import org.apache.logging.log4j.core.config.Property;
+import org.apache.logging.log4j.core.filter.BurstFilter;
+import org.apache.logging.log4j.core.layout.PatternLayout;
 import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.io.TempDir;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
 import org.parosproxy.paros.Constant;
 import org.parosproxy.paros.network.HttpMessage;
 import org.parosproxy.paros.network.HttpRequestHeader;
 import org.zaproxy.addon.commonlib.ui.ProgressPaneListener;
-import org.zaproxy.zap.utils.HarUtils;
+import org.zaproxy.zap.testutils.TestUtils;
 import org.zaproxy.zap.utils.I18N;
 
 /** Unit test for {@link HarImporter}. */
-class HarImporterUnitTest {
+class HarImporterUnitTest extends TestUtils {
 
+    private static final String APPENDER_NAME = "ZAP-TestAppender";
     private static final byte[] EMPTY_BODY = {};
 
+    private static Level originalLevel;
+    private static List<String> logMessages;
+
     @BeforeAll
     static void setup() {
         Constant.messages = mock(I18N.class);
         given(Constant.messages.getString(any())).willReturn("");
```

Comment:
I couldn't figure out how to handle the getString(String, varargs) call, so some of the tests to barf all over the console.

Reply:
Use `TestUtils`, which already does that.

Reply:
Didn't help 😞 

```console
Defaulting ZAP install dir to C:\Users\thorin\.gradle\caches\modules-2\files-2.1\org.zaproxy\zap\2.15.0\42bda528ccba07c3c8bba3121d766bbd12c10402
4051 [main] INFO  org.zaproxy.addon.exim.har.HarImporter - Setting request version to HTTP/1.1 for http://localhost/foo/one.html
4067 [main] WARN  org.zaproxy.addon.exim.har.HarImporter - null
4071 [main] ERROR org.zaproxy.zap.utils.I18N - Failed to load a message:
java.util.MissingResourceException: Can't find resource for bundle java.util.PropertyResourceBundle, key exim.progress.currentimport
	at java.util.ResourceBundle.getObject(ResourceBundle.java:564) ~[?:?]
	at java.util.ResourceBundle.getString(ResourceBundle.java:521) ~[?:?]
	at org.zaproxy.zap.utils.I18N.getStringImpl(I18N.java:183) ~[zap-2.15.0.jar:2.15.0]
	at org.zaproxy.zap.utils.I18N.getString(I18N.java:315) ~[zap-2.15.0.jar:2.15.0]
```